CMakeLists.txt

```diff
@@ -152,20 +154,35 @@ set(HTCONDORCE_DIRS ${STATE_INSTALL_DIR}/run/condor-ce ${STATE_INSTALL_DIR}/log/
   ${STATE_INSTALL_DIR}/lib/condor-ce/spool/ceview ${STATE_INSTALL_DIR}/lib/condor-ce/spool/ceview/vos
   ${STATE_INSTALL_DIR}/lib/condor-ce/spool/ceview/metrics ${STATE_INSTALL_DIR}/lib/condor-ce/execute
   ${STATE_INSTALL_DIR}/lock/condor-ce ${STATE_INSTALL_DIR}/lock/condor-ce/user
-  ${STATE_INSTALL_DIR}/lib/gratia/condorce_data ${SYSCONF_INSTALL_DIR}/logrotate.d)
+  ${STATE_INSTALL_DIR}/lib/gratia/condorce_data ${SYSCONF_INSTALL_DIR}/logrotate.d
+  ${SYSCONF_INSTALL_DIR}/condor-ce/tokens.d)
 
 foreach(dir IN ITEMS ${HTCONDORCE_DIRS})
   install(DIRECTORY DESTINATION ${dir})
   install(CODE "execute_process(COMMAND chown condor:condor ${dir})")
 endforeach(dir)
 
+set(HTCONDORCE_ROOT_DIRS ${SYSCONF_INSTALL_DIR}/condor-ce/passwords.d)
+
+foreach(root_dir IN ITEMS ${HTCONDORCE_ROOT_DIRS})
+  install(DIRECTORY DESTINATION ${root_dir})
+  install(CODE "execute_process(COMMAND chown root:root ${root_dir})")
+endforeach(root_dir)
+
 set(STICKY_HTCONDORCE_DIRS ${STATE_INSTALL_DIR}/log/condor-ce/user ${STATE_INSTALL_DIR}/lock/condor-ce/user
   ${SYSCONF_INSTALL_DIR}/logrotate.d)
 
 foreach(sticky_dir IN ITEMS ${STICKY_HTCONDORCE_DIRS})
   install(CODE "execute_process(COMMAND chmod 1777 ${sticky_dir})")
 endforeach(sticky_dir)
 
+set(PRIVATE_HTCONDORCE_DIRS ${SYSCONF_INSTALL_DIR}/condor-ce/passwords.d
+  ${SYSCONF_INSTALL_DIR}/condor-ce/tokens.d)
+
+foreach(private_dir IN ITEMS ${PRIVATE_HTCONDORCE_DIRS})
+  install(CODE "execute_process(COMMAND chmod 0700 ${private_dir})")
```

Comment:
Doesn't the `condor` user need to have access to passwords.d and/or tokens.d?

Reply:
No and yes.  `passwords.d` should be `root:root` and `0700` while `tokens.d` should be `condor:condor` and `0700`.  I _think_ that's what I have above.